<launch>    
<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <!--<rosparam file="$(find keda_move)/launch/config/move_base_teb_beiyou/costmap_converter_params.yaml" command="load" />-->
    <rosparam file="$(find ucar_controller)/param/move_base_params/move_base_teb/move_base_params.yaml" command="load" /> 
    <rosparam file="$(find ucar_controller)/param/move_base_params/move_base_teb/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find ucar_controller)/param/move_base_params/move_base_teb/costmap_common_params.yaml" command="load" ns="local_costmap" />   
    <rosparam file="$(find ucar_controller)/param/move_base_params/move_base_teb/local_costmap_params.yaml" command="load" />   
    <rosparam file="$(find ucar_controller)/param/move_base_params/move_base_teb/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find ucar_controller)/param/move_base_params/move_base_teb/teb_local_planner_params.yaml" command="load" />
<!--     
    <rosparam file="$(find ucar_controller)/param/move_base_params/move_base_teb/base_global_planner_params.yaml" command="load" /> -->
    <!--<rosparam file="$(find keda_move)/launch/config/move_base_teb_beiyou/dwa_local_planner_params.yaml" command="load" />-->
		<param name="base_global_planner" value="global_planner/GlobalPlanner" />
		<param name="controller_frequency" value="25.0" />
		<param name="controller_patience" value="3.0" />
		<param name="use_dijkstra" value="false" />

    <!-- <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/> -->

		<param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
		<param name="controller_frequency" value="25.0" />
		<param name="controller_patience" value="3.0" />

    <remap from="cmd_vel" to="cmd_vel_teb"/> 
        <remap from="scan" to="scan_0"/>
    
  </node> 
  </launch>
